libsolidity/codegen/CompilerContext.cpp

```diff
@@ -298,12 +298,14 @@ unsigned CompilerContext::baseStackOffsetOfVariable(Declaration const& _declarat
 
 unsigned CompilerContext::baseToCurrentStackOffset(unsigned _baseOffset) const
 {
-	return m_asm->deposit() - _baseOffset - 1;
+	solAssert(static_cast<unsigned>(m_asm->deposit()) > _baseOffset, "");
```

Comment:
Almost looks like this assertion is actually triggered in CI... can you add a mesage to it so that it becomes clearer?